docs/decisions/0007-reducing-complexity.md

```diff
@@ -0,0 +1,39 @@
+# Reducing Complexity
+
+## Motivation
+
+High complexity is in general a concerning problem: It can lead to untested legacy code, a high amount of bugs, bad maintainability, slow development, and so on. These decisions are concrete changes we want to make to reduce complexity.
+
+## Decisions
+
+### Hooks
+
+Decision: Hooks always start with `use` (for example `useCounter`). This applies to custom hooks. A custom hook is any function that calls other hooks, especially state and lifecycle hooks like `useEffect` or `useState`. Functions that do not call other hooks never start with `use`.
+
+Reason: See general principles (3). See [React Docs](https://beta.reactjs.org/learn/reusing-logic-with-custom-hooks#hook-names-always-start-with-use)
+
+### Contents of hooks.js
```

Comment:
A note on this: there are two parts. A. a hooks.js file should only contain hooks - I consider important because it is a convention in the React world; and B. anything else I wrote about organizing that file and were to put hooks and functions - is just an idea how we could structure it, we can change that anyway you feel like.